src/CMakeLists.txt

```diff
@@ -23,6 +23,64 @@ set(SERVER_EXE_NAME "${PROJECT_NAME}" )
 set(SERVER_OBJ_LIB_NAME ${SERVER_EXE_NAME}-object )
 set(SERVER_LIB_NAME ${SERVER_EXE_NAME}-core )
 set(SERVER_LIB_STATIC_NAME ${SERVER_LIB_NAME}-static )
+set(GRPC_CLIENT_EXE_NAME kuksa_viss_grpc_client) 
+set(gRPC_SSL_PROVIDER "package" CACHE STRING "Provider of ssl library")
+
+### grpc repository and submodule fetch
+include(FetchContent)
+## PROTOBUF ##
+FetchContent_Declare(
```

Comment:
`FetchContent_Declare` for protobuf is not needed, because grpc repo includes protobuf already